README.md

```diff
@@ -13,20 +13,18 @@ let mut mount = Mount::new();
 // Serve the shared JS/CSS at /
 mount.mount("/", Static::new(Path::new("target/doc/")));
 // Serve the static file docs at /doc/
-mount.mount("/doc/", Static::new(Path::new("target/doc/static/")));
+mount.mount("/doc/", Static::new(Path::new("target/doc/staticfile/")));
 // Serve the source code at /src/
-mount.mount("/src/", Static::new(Path::new("target/doc/src/static/lib.rs.html")));
+mount.mount("/src/", Static::new(Path::new("target/doc/src/staticfile/lib.rs.html")));
 
 Iron::new(mount).listen(Ipv4Addr(127, 0, 0, 1), 3000).unwrap();
 ```
 
-Note that `static` is a reserved keyword, so the crate will need to be imported as `extern crate "static" as static_file;`.
-
 See [`examples/doc_server.rs`](examples/doc_server.rs) for a complete example that you can compile.
 
 ## Overview
 
-static is a part of Iron's [core bundle](https://github.com/iron/core).
+staticfile is a part of Iron's [core bundle](https://github.com/iron/core).
```

Comment:
Just remove this line, that repo doesn't exist anymore and common is going to be totally redone anyway.